### ISSUE TITLE
[BUG] Scenarios/AKS-OpenAI-CogServe-Redis-Embeddings : text-davinci-002

### ISSUE THREAD

**Issue by khowling** [Azure/AKS-Landing-Zone-Accelerator#81]
**Describe the bug**
bicep deployment fails with : 

```
{"code": "DeploymentModelNotSupported", "message": "Creating account deployment is not supported by the model 'text-davinci-002'. This is usually because there are better models available for the similar functionality."}
```

Appears to be this issue ("likely related to the OpenAI deprecation announcements. I'm testing to see if gpt3-turbo can be used instead"):
Azure-Samples/azure-search-openai-demo/issues#388

#73

**Comment by khowling** [comment]
@mosabami , I cannot get past this one, I dont have enough knowledge to replace models.

**Comment by mosabami** [comment]
fixed with #71
